demos/modals/Modals.module.scss

```diff
@@ -18,35 +18,25 @@
   border-right: 1px solid var(--theme-border);
   box-sizing: content-box;
   transition: left 0.3s;
+  animation: slideInAnimation 0.3s;
```

Comment:
`0.3s` feels like an arbitrary number, lets keep it to `200ms` which is the minimal amount (180ms) for the eye to notice.

Reply:
i left the value as i found it, i just moved it around. i can make the change though.

Reply:
Understood, probably worthwhile!